src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -36,15 +37,28 @@ export const useFormSend = () => {
 
         const { estimatedFees, feeType, address: to } = data
         if (adapterType === ChainTypes.Ethereum) {
+          if (!supportsETH(wallet)) throw new Error(`useFormSend: wallet does not support ethereum`)
           const fees = estimatedFees[feeType] as chainAdapters.FeeData<ChainTypes.Ethereum>
-          const gasPrice = fees.chainSpecific.gasPrice
-          const gasLimit = fees.chainSpecific.gasLimit
+          const {
+            chainSpecific: { gasPrice, gasLimit, maxFeePerGas, maxPriorityFeePerGas }
+          } = fees
           const address = isEthAddress(to) ? to : ((await ensLookup(to)).address as string)
+          const useEIP1559Fees =
```

Comment:
Nitpick: `useXYZ` is usually used for naming hooks/hook-ish variables in React. Would be happier with something like `withBaseFee` or `withEIP1559Fees`

Reply:
good point; changed to `shouldUseEIP1559Fees`.